syntax_and_semantics/capturing_blocks.md

```diff
@@ -15,7 +22,14 @@ proc.call(1) #=> 2
 
 The above code captures the block of code passed to `int_to_int` in the `block` variable, and returns it from the method. The type of `proc` is [Proc(Int32, Int32)](http://crystal-lang.org/api/Proc.html), a function that accepts a single `Int32` argument and returns an `Int32`.
 
-In this way a block can be saved as a callback:
+The simple example covers all requirements for capturing a block:
+- def int_to_int(__&block__ : Int32 -> Int32) - specify it as a method's block argument
```

Comment:
Code shoud be wrapped in backticks and you shouldn't need to highlight specific parts.